### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by SunnyAsar**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,21 +1,428 @@
...
+          </li>
+          <li class="nav-item mx-1">
+            <i class="fab fa-linkedin-in"></i>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </main>
+
+  <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo"
```
Comment (zlayabekrija) [inline index.html]: JS and JQuery aren't necessary in this project, you can remove them.

File: index.html [hunk 2]
```diff
@@ -1,21 +1,428 @@
...
+          </path>
+        </g>
+      </svg>
+    </div>
+    <div class="pull-right ">
```
Comment (zlayabekrija) [inline index.html]: this is bootstrap 3 syntax, but you are using bootstrap for, correct syntax for this is order-number(1,2,3....)
please adjust it.

File: index.html [hunk 3]
```diff
@@ -1,21 +1,428 @@
...
-    integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+    crossorigin="anonymous">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr"
+    crossorigin="anonymous">
   <link rel="stylesheet" href="assets/css/styles.css">
 </head>
+
 <body>
+  <nav class="navbar navbar-dark sticky-top  bg-danger text-white pt-2 pt-sm-4">
```
Comment (zlayabekrija) [inline index.html]: fixed will work here

File: index.html [hunk 4]
```diff
@@ -1,21 +1,428 @@
...
+                      institution took them back," said Senator Mike Lee, a Utah Republican.</p>
+                    <hr>
+                    <h6>Trump’s Climate Denial Gets More Ridiculous By The Day</h6>
+                    <hr>
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div class="col-md-7">
```
Comment (zlayabekrija) [inline index.html]: there are 3 break points in this project (sm,md,lg), please adjust and use them.

File: index.html [hunk 5]
```diff
@@ -1,21 +1,428 @@
...
+                    </li>
+                  </ul>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="col-md-4 ">
```
Comment (zlayabekrija) [inline index.html]: same applies here, please use three break points.

**Review by zlayabekrija — CHANGES_REQUESTED** [review]
Excellent first try, but there are many things to be done here.
Please make your project according to original Newsweek. Three break points, different order on each of breaking points, your medium (md) and large (lg) are absolutely the same, unlike the original.
Please add some more content (featured stories are missing) and enhance footer.
After all of these adjustments are made, please submit a new review request.
Happy coding :) 
P.S.
Feel free to contact me on slack if you need any help.

File: index.html [hunk 6]
```diff
@@ -1,21 +1,659 @@
...
+                    <hr>
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div class="col-xs-12 col-md-7">
+              <div class="container-fluid">
+                <div class="row">
+                  <h5 class="text-left text-secondary h6 font-weight-bold">TOP STORIES</h5>
```
Comment (zlayabekrija) [inline index.html:104]: use `order-first order-1-lg` here in order to have top stories as a first ,and second where needed.
Follow this example in other cases please.

**Review by zlayabekrija — CHANGES_REQUESTED** [review]
Great second try, many more things to be done.
On the smallest screen size, the top story is first, then more stories, please use chrome's inspector tool to have a clearer image, then featured stories (first in your case). 
Same applies on medium screen size.
Please check bootstrap documentation more on this matter.
I gave you on example inside of your code, please follow it and adjust accordingly.
For any further explenation please feel free to contact me on the slack.
Submit new review request when ready.
Happy coding :)

File: index.html (hunk 6 above)
Comment (HenryTabima) [reply index.html:104]: thanks! we didn't notice the reordering :sweat_smile:

**Review by zlayabekrija — APPROVED** [review]
Good work, thumbs up from me. 👍

**Pull request merged**